activerecord/test/schema/postgresql_specific_schema.rb

```diff
@@ -19,6 +19,7 @@
   execute <<_SQL
     CREATE TABLE defaults (
     id serial primary key,
+    four_bytes_int integer default 0,
```

Comment:
We shouldn't need to modify the schema in any way. You can use any integer column from any table. Just pick one.